reporter/src/test/kotlin/reporters/freemarker/FreeMarkerTemplateProcessorTest.kt

```diff
@@ -628,4 +630,30 @@ class FreeMarkerTemplateProcessorTest : WordSpec({
             results should beEmpty()
         }
     }
+
+    "getPackage" should {
+        "return the correct package for the given ID" {
+            val id = Identifier.EMPTY.copy(name = "test-package", version = "1.0.1")
+            val pkg = Package.EMPTY.copy(
+                id = id,
+                cpe = "cpe:2.3:a:test:${id.name}:${id.version}:-:-:-:-:-:-:-"
+            )
+            val analyzerResult = ORT_RESULT.analyzer!!.result.copy(packages = sortedSetOf(CuratedPackage(pkg)))
+            val analyzerRun = ORT_RESULT.analyzer!!.copy(result = analyzerResult)
+
+            val input = ReporterInput(ORT_RESULT.copy(analyzer = analyzerRun))
```

Comment:
Injecting a mock OrtResult would strip a bit code redundancy, what do you think?

Reply:
True, but all other tests in this class use real data structures.

@mnonnenmacher mentioned once that he has the idea for a DSL to generate test data structures. This would be a good case for such a tool.